examples/experimental/bezier/app.js

```diff
@@ -98,41 +103,13 @@ function layoutGraph(graph) {
 class App extends Component {
   constructor(props) {
     super(props);
-    this.state = {
-      viewport: {
-        width: 500,
-        height: 500
-      }
-    };
-  }
-
-  componentDidMount() {
-    window.addEventListener('resize', this._resize.bind(this));
-    this._resize();
-  }
-
-  _resize() {
-    this.setState({
-      viewport: {
-        width: window.innerWidth,
-        height: window.innerHeight
-      }
-    });
   }
 
   render() {
-    const {viewport} = this.state;
-    const {width, height} = viewport;
-
     const {data = layoutGraph(SAMPLE_GRAPH)} = this.props;
 
     const view = new OrthographicView({
-      left: -width / 2,
-      top: -height / 2,
-      right: width / 2,
-      bottom: height / 2,
-      near: 0,
-      far: 100
+      controller: OrthographicController
```

Comment:
I'm not sure why I still need to specify the controller here even OrthographicView already has it as default controller at line 8 in orthographic-controller.js

Reply:
A `View` doesn't get a controller by default. You should however be able to just specify `controller: true` if you are fine with the default controller type for that view.

Does that work?

Reply:
Thanks for the clarification. I asked since I didn't understand this API

Reply:
Setting `controller: true` should work - and needs to work for JSONLayers to be able to use this.